src/planscape/collaboration/utils.py

```diff
@@ -0,0 +1,110 @@
+from collaboration.models import Collaborator, Permissions, Role
+from planning.models import PlanningArea, Scenario
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+
+
+User = get_user_model()
+
+
+def is_creator(user: User, planning_area: PlanningArea):
+    return planning_area.user.pk == user.pk
+
+
+def has_access(user_id, model, permission):
```

Comment:
I like this, but could we change the name to something more like `has_permission` or `check_for_permission`, just to clarify that we're looking up a permission?